.github/workflows/test.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - 6-docker-compose-instead-of-docker-compose
+      - '[0-9]*-*[a-zA-Z]*'  # regex should match typical issue branches names
```

I thing better branch naming would be `feature/*` for each issue branch

```suggestion
      - 'feature/*'
```